source/api/auth/0.9/index.md

```diff
@@ -93,17 +94,17 @@ The Description Resource _MUST_ include a service using the following template:
 }
 {% endhighlight %}
 
-Where the `@id` field _MUST_ be present and contains the URI that the client should load to allow the user to authenticate. The value of `profile` _MUST_ be `http://iiif.io/api/auth/{{ page.major }}/login`, allowing clients to understand the use of the service. The `label` property _SHOULD_ be present and its value is to be shown to the user to initiate the loading of the authentication service.  The `service` field _MUST_ be present and will contain related services described below.
+Where the `@id` field _MUST_ be present and contains the URI that the client should load to allow the user to authenticate. The `@context` field _MUST_ be present with the value `http://iiif.io/api/auth/{{ page.major }}/context.json`, specifying the context to resolve the keys into RDF if necessary. The value of `profile` _MUST_ be `http://iiif.io/api/auth/{{ page.major }}/login`, allowing clients to understand the use of the service. The `label` property _SHOULD_ be present and its value is to be shown to the user to initiate the loading of the authentication service.  The `service` field _MUST_ be present and will contain related services described below.
 
```

Comment:
Sure. Some people have said I use too many small but verbose constructions like this, but if it makes it clearer...